articles/libraries/lock/v10/index.md

```diff
@@ -61,14 +61,24 @@ If you are using browserify or webpack to build your project and bundle its depe
 
 ### 1. Initializing Lock
 
-First, you'll need to initialize a new `Auth0Lock` object, and provide it with your Auth0 client ID (the unique client ID for each Auth0 client app, which you can get from the [management dashboard](${manage_url})) and your Auth0 domain (i.e. `jeffstest.auth0.com`).
+First, you'll need to initialize a new `Auth0Lock` object (or an `Auth0LockPasswordless` object, if you are going to be implementing passwordless authentication). Then, you'll have to provide your Auth0 client ID (the unique client ID for each Auth0 client app, which you can get from the [management dashboard](${manage_url})) and your Auth0 domain (i.e. `jeffstest.auth0.com`). Both `Auth0Lock` and `Auth0LockPasswordless` have the same public methods.
```

Comment:
why use `jeffstest.auth0.com` and not `${account.namespace}` like in the script?

Reply:
I'd like to make this maybe something like `example.auth0.com` - it's supposed to be an example of what an Auth0 domain looks like, and if the user is not signed in, we won't get metadata, so they will just get the placeholder text, which will not be an example of what an Auth0 domain looks like.